src/windows/map.c

```diff
@@ -290,18 +292,20 @@ static void window_map_mouseup()
 
 	case WIDX_LAND_TOOL_SMALLER:
 		--RCT2_GLOBAL(RCT2_ADDRESS_LAND_TOOL_SIZE, sint16);
+		land_tool_limit=1;
 
-		if (RCT2_GLOBAL(RCT2_ADDRESS_LAND_TOOL_SIZE, sint16) < 1)
-			RCT2_GLOBAL(RCT2_ADDRESS_LAND_TOOL_SIZE, sint16) = 1;
+		if (RCT2_GLOBAL(RCT2_ADDRESS_LAND_TOOL_SIZE, sint16) < land_tool_limit)
+			RCT2_GLOBAL(RCT2_ADDRESS_LAND_TOOL_SIZE, sint16) = land_tool_limit;
 
 		window_invalidate(var_w);
 		break;
 
 	case WIDX_LAND_TOOL_LARGER:
 		++RCT2_GLOBAL(RCT2_ADDRESS_LAND_TOOL_SIZE, sint16);
+		land_tool_limit=64;
```

Comment:
It has been done that way for the land tool and the water tool. That's why I did it this way. I can change it if that's better.